modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -110,6 +110,12 @@ Ciphers and the minimum TLS version of the configured security profile are refle
 * `Strict`: does not allow routes to claim the same host name across namespaces.
 * `InterNamespaceAllowed`: allows routes to claim different paths of the same host name across namespaces.
 
+`wildcardPolicy` describes how routes with wildcard policies are handled by the Ingress Controller.
+
+* `WildcardsAllowed`: Indicates routes with any wildcard policy are admitted by the Ingress Controller.
+
+* `WildcardsDisallowed`: Indicates only routes with a wildcard policy of `None` are admitted by the Ingress Controller. Updating `wildcardPolicy` from `WildcardsAllowed` to `WildcardsDisallowed` causes admitted routes with a wildcard policy of subdomain to stop working. These routes must be recreated to a wildcard policy of `None` to be readmitted by the Ingress Controller. `WildcardsDisallowed` is the default setting.
```

Comment:
... causes admitted routes with  a wildcard policy of `Subdomain` to stop working. Using `Subdomain` instead subdomain here.
see also
```
      --wildcard-policy='': Sets the WilcardPolicy for the hostname, the default is "None". valid values are "None" and
"Subdomain"
```